src/main/java/com/iexec/sms/authorization/AuthorizationService.java

```diff
@@ -140,7 +140,7 @@ public String getChallengeForSetAppRuntimeSecret(String appAddress,
                 Hash.sha3String(secretValue));
     }
 
-    public String getChallengeForSetAppRequesterRuntimeSecretCount(
+    public String getChallengeForSetAppRequesterAppRuntimeSecretCount(
```

Comment:
You can update ~~AppRequester~~ -> Requester everywhere :)

Reply:
+1

Reply:
Done there: https://github.com/iExecBlockchainComputing/iexec-sms/commit/a6e17fd41d18dda94b9133dbb34ecbbb0696c336